### DISCUSSION TITLE
[Feature] Create Shared Album with shared Images

### DISCUSSION THREAD

**Discussion by Moosbee** [immich-app/immich#10057] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The abilaty to add images wich got shared with you to other shared albums.

The accouts used in the new Album will likly also have access to the new album

My Problem, we had a vacation and all took a lot of immages(16000) on different accounts, now we created a gigantic shared album, and now I want to create a smaller album from the big one, I can select all the images but when I go to "add to shared album" only my images and my partners one get added to the new album. Moreover when I try to add one manualy, Immich says "Image already in Album" (it isn't) and also doesn't add it.

What I want to have is the to add images wich got shared with me to an other shared albums

I will happily help implement a solution 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bunicb** [comment]
I have the same problem. Me and my partner have some shared albums which are growing rapidly.
We want to create separate albums with subsets of images, either for slideshows or for printing, but can only add own images, and there's no apparent error when trying to add shared images.

This could be a setting to allow sharing images shared with me, either on a server level, or an album level.

**Comment by bo0tzz** [comment]
#5643
